kotlinx-coroutines-core/jvm/test/linearizability/LockFreeListLCStressTest.kt

```diff
@@ -44,29 +44,11 @@ class LockFreeListLCStressTest : TestBase() {
     private fun Any.isSame(value: Int) = this is Node && this.value == value
 
     @Test
-    fun testAddRemoveLinearizability() {
-        val options = StressOptions()
-            .iterations(100 * stressTestMultiplierSqrt)
-            .invocationsPerIteration(1000 * stressTestMultiplierSqrt)
-            .threads(3)
-        LinChecker.check(LockFreeListLCStressTest::class.java, options)
-    }
-
-    private var _curElements: ArrayList<Int>? = null
-    private val curElements: ArrayList<Int> get() {
-        if (_curElements == null) {
-            _curElements = ArrayList()
-            q.forEach<Node> { _curElements!!.add(it.value) }
-        }
-        return _curElements!!
-    }
-
-    override fun equals(other: Any?): Boolean {
-        other as LockFreeListLCStressTest
-        return curElements == other.curElements
-    }
+    fun testAddRemoveLinearizability() = LCStressOptionsDefault().check(this::class)
```

Comment:
Note, that previous version of LinCheck was running this test with 5 actors per thread and it turns out this was quite important to increase probability of catching certain problems. I'd recommend to leave it this way for this particular test.

Reply:
OK.